src/nfts/NftsInfoCarousel.tsx

```diff
@@ -0,0 +1,350 @@
+                    `${nft.contractAddress}-${nft.tokenId}`
+                  }
+                  // If there's no nft metadata, show a red info icon instead
+                  error={!nft.metadata}
+                  nft={nft}
+                />
+              </View>
+            )
+          })}
+        </ScrollView>
+      </View>
+    )
+  }
+
+  // Display error screen no Nfts provided
+  if (nfts.length === 0) {
+    return <NftsLoadErrorScreen />
+  }
+
+  return (
+    <SafeAreaView edges={['top']} style={styles.safeAreaView} testID="NftsInfoCarousel">
+      <ScrollView>
+        {activeNft.metadata && (
+          <FastImage
+            testID={`NftsInfoCarousel/NftImage-${activeNft.contractAddress}-${activeNft.tokenId}`}
+            style={[
+              { height: scaledHeight },
+              styles.mainImage,
+              // Put a border radius on the image when loading to match placeholder
+              isLoading && styles.borderRadius,
+            ]}
+            source={{
+              uri: activeNft.media.find((media) => media.raw === activeNft.metadata?.image)
+                ?.gateway,
+            }}
+            onLoad={(e) => {
+              setScaledHeight(
+                scaleImageHeight(e.nativeEvent.width, e.nativeEvent.height, variables.width)
+              )
+            }}
+            onLoadEnd={() => setIsLoading(false)}
+            onError={() =>
+              Logger.error(
+                TAG,
+                `Error loading Nft image for Nft contractAddress: ${activeNft.contractAddress} tokenId: ${activeNft.tokenId}`
+              )
+            }
+            resizeMode={FastImage.resizeMode.contain}
+          >
+            {isLoading && <MainImagePlaceholder />}
+          </FastImage>
+        )}
+        {/* This could happen if the indexer is experiencing issues with particular nfts */}
+        {!activeNft.metadata && (
+          <View style={styles.nftImageLoadingErrorContainer}>
+            <Text style={{ color: colors.light }}>{t('nftInfoCarousel.nftImageLoadError')}</Text>
+          </View>
+        )}
+
+        {nfts.length > 1 && <NftImageCarousel />}
+        <View style={styles.sectionContainer}>
+          <Text style={styles.title}>{activeNft.metadata?.name}</Text>
+        </View>
+        {activeNft.metadata?.description && (
+          <View style={styles.sectionContainer}>
+            <Text style={styles.subSectionTitle}>{t('nftInfoCarousel.description')}</Text>
+            <Text style={styles.text}>{activeNft.metadata?.description}</Text>
+          </View>
+        )}
+        {activeNft.metadata?.attributes && (
+          <View style={styles.sectionContainer}>
+            <Text style={styles.subSectionTitle}>{t('nftInfoCarousel.attributes')}</Text>
+            {activeNft.metadata?.attributes.map((attribute, index) => (
+              <View key={index} style={styles.attributesContainer}>
```

Comment:
Looking at the [OpenSea's standard](https://docs.opensea.io/docs/metadata-standards#date-traits) it looks like there might be cases where there is no trait_type and value isn't unique so this might be the best we can do.
> If you don't want to have a trait_type for a particular trait, you can include just a value in the trait and it will be set as a generic property
```
{
  "value": "Happy"
}
```